Assets/HoloToolkit/Input/Scripts/Utilities/SetGlobalListener.cs

```diff
@@ -13,33 +13,23 @@ public class SetGlobalListener : MonoBehaviour
     {
         private void OnEnable()
         {
-            StartCoroutine(AddGlobalListener());
+            InputManager.Instance.AddGlobalListener(gameObject);
```

Comment:
We should probably still have an `Instance != null` check here, just in case.

Reply:
We would want it to throw the null ref.  If it's not found by then, it means the object either isn't in the scene or disabled. 

The Singleton updates should have fixed the issues we were having with Awake not being called in time.